app/lib/ldap_authenticatable_tiny/strategy.rb

```diff
@@ -14,10 +14,13 @@ class LdapAuthenticatableTiny < Authenticatable
       # https://github.com/plataformatec/devise/blob/master/lib/devise/models/authenticatable.rb
       def authenticate!
         email = authentication_hash[:email]
+        should_use_mock_ldap = ::EnvironmentVariable.is_true('USE_TEST_PALS_LDAP')
+        ldap_service = should_use_mock_ldap ? TestPalsMockLDAP : Net::LDAP
```

Comment:
This is small but this is a Ruby class not an instance so maybe `ldap_class`?

Reply:
👍